assets/dev/js/editor/regions/panel/commands/internal/open-default.js

```diff
@@ -3,6 +3,9 @@ import CommandInternalBase from 'elementor-api/modules/command-internal-base';
 export class OpenDefault extends CommandInternalBase {
 	apply() {
 		$e.route( elementor.documents.getCurrent().config.panel.default_route );
+
```

Comment:
It does not help for the current problem.
if the parent command will return also a promise it will help, but for now when returning a promise from the parent command, the tests fail.

so the event a quick fix for the current problem.